hydra_base/util/hdb.py

```diff
@@ -376,6 +376,10 @@ def create_default_units_and_dimensions():
                 # If ID is specified
                 new_dimension.id = json_dimension["id"]
 
+            if "is_default" in json_dimension:
```

Comment:
didn't we agree to remove the is_default column?

Reply:
remoed it